### DISCUSSION TITLE
Can we scale and rotate basic shapes as well once drawn

### DISCUSSION THREAD

**Discussion by prashanthremis** [burhanrashid52/PhotoEditor#447] [Q&A]
Can we scale and rotate basic shapes as well once drawn in canvas like the rect, ellipse.

**Comment by burhanrashid52** [comment]
We are drawing brush directly on one canvas. So currently it's not possible. Maybe you can have a separate screen to draw on the canvas and then convert that canvas as an image and add in PhotoEditor.
